src/services/ws.js

```diff
@@ -91,6 +91,20 @@ class WebsocketService {
 
         return { lobby, player };
     }
+
+    /**
+     * Send notifications to everyone
+     * 
+     * @param lobbyId Room id
+     * @param message Message to notify
+     */
+    broadcastLobby(lobbyId, message) {
+        if (typeof message !== "string") {
+            throw new Error("Message is not a string");
+        }
+
+        this.ws.to(lobbyId).emit("notify lobby", message);
```

Comment:
C'est vague comme type d'évènement, à changer. Du coup le nom de la méthode aussi.

Supprimer le `this.ws.to(client.data.lobby.id).emit("newPlayer", client.data.player.name);` (ligne 65) car la notif est envoyée en double. Sinon tu te mets d'accord avec @Tomansion